docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
+has been configured, click the ``+ Add Attached Storage`` button in
+the upper-right corner. This is illustrated in the following image.
 
-.. image:: _static/noaacsp_cluster_3.png
+.. image:: _static/noaacsp_lustre_5.png
 
-Next the partitions for the cluster may be defined. A partition
-configuration for the aforementioned C48 atmosphere forecast-only
-application is illustrated in the figure below. Note that the instance
-type beneath ``Controller Settings`` and ``Partitions`` must be
-identical. Other configurations are not supported by the
-global-workflow team. Once the partitions are configured, click the
-``+ Add Partition`` button in the upper-right corner.
+******************************
+Attach CSP Lustre Filesystem
+******************************
 
-.. image:: _static/noaacsp_cluster_4.png
+Now we need to attach the defined filesystem to our cluster.
+Go back to our noaa.parallel.works web-site, and click `Cluster`
+as shown in figuer below, then select the cluster "AWS EPIC Wei CentOS example"
+(it should be your own cluster) cluster as show in red-box.
+Note, one can remove/delete this cluster if no longer needed by
+click the trash-can shown in the red-circle at right.
 
-For the storage do be allocated for the global-workflow application it
-is suggested that the ``Mount Point`` be ``/lustre``. Once the storage
-has been configured, click the ``+ Add Attached Storage`` button in
-the upper-right corner. This is illustrated in the following image.
+.. image:: _static/noaacsp_filesystem_1.png
+
+When get into the cluster page, click the `Definition` in the top menu as
+in the red-box. When finished, remeber to clicke `Save Changes` to save
+the changes.
+
+.. image:: _static/noaacsp_filesystem_2.png
+
+Scrow down to the bottom, and click `Add Attached Filesystems` as in the red-circle.
 
-.. image:: _static/noaacsp_cluster_5.png
+.. image:: _static/noaacsp_filesystem_3.png
 
-Finally, the following illustrates a JSON version of the cluster
-configuration created from the steps above. When opening issues
-related to the NOAA CSP global-workflow applications please include
-the JSON content.
+After click `Add Attached Filesystems`, one `Attached Filesystems settings` will appear.
 
-.. image:: _static/noaacsp_cluster_6.png
+1. In the `Storage` box, select the lustre filesystem defined above, as in red-arrow 1.
+2. In the `Mount Point` box, name it `/lustre` (the common and default choice) as pointed by red-arrow 2.
+   If you choose a different name, make sure to make the Global-Workflow setup step
+   use the name chosen here.
+
+If you have a `S3 bucket`, one can attached as:
+
+3. In the `Storage` box, select the bucket you want to use, as in red-arrow 3.
+4. In the `Mount Point` box, name it `/bucket` (the common and default choice) as pointed by red-arrow 4.
+
+.. image:: _static/noaacsp_filesystem_4.png
+
+Always remember to click `Save Changes` after making any changes to the cluster.
```

Comment:
```suggestion
1. In the `Storage` box, select the `/lustre` filesystem defined above, as indicated by red arrow 1.
2. In the `Mount Point` box, name it `/lustre` (the common and default choice) as indicated by red arrow 2.
   If you choose a different name, make sure that the Global-Workflow setup step
   uses the name chosen here.

If you have a `S3 bucket`, it can be attached as follows:

3. In the `Storage` box, select the bucket you want to use, as in red-arrow 3.
4. In the `Mount Point` box, name it `/bucket` (the common default choice) as indicated by red arrow 4.

.. image:: _static/noaacsp_filesystem_4.png

Always remember to click `Save Changes` after making any changes to the cluster.
```